backend/app/requests/adequacy.py

```diff
@@ -69,9 +71,10 @@ def adequacy_request(app, flask_request, engine):
     if 'method' not in request:
         raise InvalidFormat(message='Invalid format. Could not find method.')
 
-    provider_addresses = request['providers']
+    provider_locations = request['providers']
     service_area_ids = request['service_area_ids']
     measurer_methods = config.get('measurer').keys()
+    dataset_hint = request.get('dataset_hint')
```

Comment:
Can you default to None maybe? In case we expose the API separately, it is good to have a more 'normal' behavior

Reply:
Done.